kompose.spec

```diff
@@ -1001,6 +1033,9 @@ export LDFLAGS=%{ldflags}
 %endif
 
 %changelog
+* Fri Jul 21 2017 Suraj Deshmukh <surajssd009005@gmail.com> - 0-0.1.git253ec14
```

Comment:
@surajssd Fri Jul 21 2017 Suraj Deshmukh <surajssd009005@gmail.com> - 1.0.0-git253ec14 shouldn't this like this?